terracumber_config/tf_files/SUSEManager-Test-Hexagon-NUE.tf

```diff
@@ -132,30 +131,34 @@ module "cucumber_testsuite" {
       }
     }
     server_containerized = {
+      image = "slemicro55o"
       provider_settings = {
         mac = "aa:b2:93:01:00:51"
-        memory = 16384
+        vcpu = 8
+        memory = 32768
```

Comment:
How comes you double the RAM?

Reply:
I've just ported all the changes already present in https://github.com/SUSE/susemanager-ci/blob/master/terracumber_config/tf_files/SUSEManager-Head-NUE.tf